README.md

```diff
@@ -1,5 +1,11 @@
 # OpenID Connect Doctor (AMOS SS 2022)
 
+<p align="center">
+<img src="public/images/logo-amos.png" width="430" height="300">
+</p>
+
```

Comment:
Please use markdown-syntax in the readme.md

Reply:
It is too large with the following syntax:  ![OIDC Doctor](public/images/logo-amos.png)
You can change the size only with HTML syntax in markdown.